src/test/java/resources/NasaResourceTest.java

```diff
@@ -0,0 +1,92 @@
+package resources;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+
+import jakarta.ws.rs.core.Response;
+import java.util.Arrays;
+import java.util.List;
+import millom.sandbox.pojos.Descriptions;
+import millom.sandbox.pojos.Sol;
+import millom.sandbox.pojos.Weather;
+import millom.sandbox.service.WeatherService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.mockito.runners.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.class)
+public class NasaResourceTest {
+  @Mock
+  private WeatherService weatherServiceMock;
+  @InjectMocks
+  private NasaResource nasaResource = new NasaResource();
+
+  @BeforeEach
+  private void setUp() throws Exception{
+    MockitoAnnotations.initMocks(this);
```

Comment:
Not needed no?

Reply:
I am removing @RunWith(MockitoJUnitRunner.class) for now. Because it is not working as expected. Having only MockitoAnnotations.initMocks(this);